src/api/component.js

```diff
@@ -127,10 +105,11 @@ export default class extends HTMLElement {
   static get observedAttributes () {
     const attrsOnCtor = this.hasOwnProperty($ctorObservedAttributes) ? this[$ctorObservedAttributes] : [];
 
-    const props = getPropsMap(this);
-    const attrsFromLinkedProps = Object.keys(props).map(key => {
-      const { attribute } = props[key];
-      return attribute === true ? dashCase(key) : attribute;
+    const propDefs = getPropsMap(this);
+    // todo: with Object.keys we skip symbol properties!
```

Comment:
Good point, it *should*. The tests currently pass because we're transpiling, I believe, but it doesn't appear that it'd work in ES2015. If it takes too long for you to test and fix in this PR, maybe raise a separate issue and fix in that.